xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -349,6 +353,9 @@ bool CRenderManager::Configure()
       return false;
   }
 
+  // configure the renderer to crop if necessary
+  m_pRenderer->ConfigureCropping(m_cropBottom, m_cropTop, m_cropLeft, m_cropRight);
```

Comment:
the same is true for width, height, etc. following your suggestion we would need extra methods for all of those.